travis-ci/test_inside_docker.sh

```diff
@@ -47,5 +47,6 @@ cp /etc/condor/config.d/99-local.conf /etc/condor-ce/config.d/99-local.conf
 export _condor_CONDOR_CE_TRACE_ATTEMPTS=60
 
 # Ok, do actual testing
+INSTALL_STR="--install ${PACKAGES//,/ --install }"
 echo "------------ OSG Test --------------"
-osg-test -vad --hostcert --no-cleanup --install ${PACKAGE}
+osg-test -vad --hostcert --no-cleanup ${INSTALL_STR}
```

Comment:
what struck me here is that you could still leave `--install` in the `osg-test` command line, something like `--install ${PACKAGES//,/ }`, but as discussed with @brianhlin, these things just depend on the mood your in for any given day.

How are you feeling today -- Cloudy with a chance of meatballs?  Soup's on!